<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>266</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Bahnschrift</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Pinyin Tones</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
     <property name="spacing">
      <number>20</number>
     </property>
     <item row="5" column="0">
      <widget class="QLineEdit" name="shortcut_edit">
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>14</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="record_button">
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Start Recording</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="shortcut_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Record 2 buttons to set as your 'pause app' shortcut:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QComboBox" name="language_box">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="duplicatesEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="keyboard_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>~~~        Choose your desired keyboard:       ~~~</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QPushButton" name="confirm_button">
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
